content/blog/2019-05-22-ramlegacy.md

```diff
@@ -0,0 +1,51 @@
+---
+slug: "ramlegacy"
+title: "ramlegacy: a package for RAM Legacy Database"
+package_version: 0.2.0
+authors:
+  - Carl Boettiger
+  - Kshitiz Gupta
+date: '2019-05-22'
+categories: blog
+topicid:
+tags:
+- Software Peer Review
+- R
+- community
+- software
+- packages
+- ramlegacy
+- marine-biology
+- fisheries
+- stock-assessment
+---
+
+## Introduction
+
+`ramlegacy` is a new R package to download, cache and read in all the different versions of the  RAM Legacy Stock Assessment Database, a public database containing time-series and biological metrics from over 300 globally-distributed stock assessments. The package accomplishes all this by:
+
+* Providing a function `download_ramlegacy()`, to download all the available versions of the RAM Database and cache them on the user’s computer in a location provided by the [rappdirs](https://cran.r-project.org/web/packages/rappdirs/index.html) package. This way once a version has been downloaded it doesn’t need to be re-downloaded for subsequent analysis.
+* Supporting reading in specified tables or all tables from a cached version of the database through the function `load_ramlegacy()`
+* Providing a function `ram_dir()` to view the path of the rappdirs location where the downloaded database was cached.
+
+The primary motivation behind developing the caching behavior of the package was to save the user time and effort spent in re-downloading different versions of the database as part of any future analysis.
+
+`ramlegacy` been tested and peer reviewed thanks to rOpenSci and is now released on CRAN, making this an ideal time to present the package to the world. For installation and the code, see its [home](https://github.com/ropensci/ramlegacy) in rOpenSci, and the package documentation at its [website](https://ropensci.github.io/ramlegacy/).
+
+In this post, we’ll talk a little bit about the package development process we went through for `ramlegacy`.
+
+## Package Development Process
+
+When we first started developing `ramlegacy` in Sep 2018, the different versions of the database were all hosted on a Wordpress website created by the RAM maintainers. The package back then was using that Wordpress website as download source for all those versions. However, shortly after v4.3 was released the older versions (v1.0, v2.0, v2.5, v3.0) became unavailable on the Wordpress website. We believed it was important for the package to continue providing access to the aforementioned older versions because, among other reasons, it would enable the users to reproduce dozens of high profile papers based on those older versions; for example [this one](https://doi.org/10.1126/science.aau1758) recently appearing in Science uses the v3.0 data. Consequently, we decided to host the older versions of the database at a GitHub assets [repo](https://github.com/kshtzgupta1/ramlegacy-assets) making sure we appropriately acknowledged and licensed the data.
```

Comment:
```suggestion
When we first started developing ramlegacy in September 2018, the different versions of the database were all hosted on a WordPress website created by the RAM maintainers. The package back then was using that WordPress website as download source for all those versions. However, shortly after v4.3 was released the older versions (v1.0, v2.0, v2.5, v3.0) became unavailable on the WordPress website. We believed it was important for the package to continue providing access to the aforementioned older versions because, among other reasons, it would enable the users to reproduce dozens of high profile papers based on those older versions; for example [this one](https://doi.org/10.1126/science.aau1758) recently appearing in Science uses the v3.0 data. Consequently, we decided to host the older versions of the database at a GitHub assets [repo](https://github.com/kshtzgupta1/ramlegacy-assets) making sure we appropriately acknowledged and licensed the data.
```

Suggested changes that can be accepted as a commit
- Wordpress -> WordPress
- Sep -> September
- `ramlegacy` -> ramlegacy